Time:2016/7/27 9:38:08-Trace>>OnInit error:ServiceStack.Redis.RedisException: could not connect to redis Instance at 127.0.0.1:6379 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 127.0.0.1:6379
   在 System.Net.Sockets.Socket.Connect(IPAddress[] addresses, Int32 port)
   在 System.Net.Sockets.Socket.Connect(String host, Int32 port)
   在 ServiceStack.Redis.RedisNativeClient.Connect()
   --- 内部异常堆栈跟踪的结尾 ---
   在 ServiceStack.Redis.RedisNativeClient.Connect()
   在 ServiceStack.Redis.RedisNativeClient.AssertConnectedSocket()
   在 ServiceStack.Redis.RedisNativeClient.SendCommand(Byte[][] cmdWithBinaryArgs)
   在 ServiceStack.Redis.RedisNativeClient.SendExpectCode(Byte[][] cmdWithBinaryArgs)
   在 ZyGames.Framework.Redis.RedisConnectionPool.ProcessTrans(RedisClient client, String[] watchKeys, Func`1 processFunc, Action`1 transFunc, Action`2 errorFunc)
   在 ZyGames.Framework.Redis.RedisConnectionPool.<>c__DisplayClass10.<ProcessTrans>b__e(RedisClient client)
   在 ZyGames.Framework.Redis.RedisConnectionPool.Process(Action`1 func)
   在 ZyGames.Framework.Redis.RedisConnectionPool.ProcessTrans(String[] watchKeys, Func`2 processFunc, Action`1 transFunc, Action`2 errorFunc)
   在 ZyGames.Framework.Redis.RedisConnectionPool.InitRedisInfo()
   在 ZyGames.Framework.Redis.RedisConnectionPool.Initialize(ICacheSerializer serializer)
   在 ZyGames.Framework.Game.Runtime.GameEnvironment.Start(EnvironmentSetting setting, CacheSetting cacheSetting)
   在 ZyGames.Framework.Game.Runtime.RuntimeHost.OnStart()


Time:2016/7/27 9:38:32-Trace>>OnInit error:System.Exception: Not connect to the database server "localhost" database "NewbeeConfig". ---> MySql.Data.MySqlClient.MySqlException: Authentication to host 'localhost' for user 'root' using method 'mysql_native_password' failed with message: Unknown database 'NewbeeConfig' ---> MySql.Data.MySqlClient.MySqlException: Unknown database 'NewbeeConfig'
   在 MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   在 MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket()
   --- 内部异常堆栈跟踪的结尾 ---
   在 MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.AuthenticationFailed(Exception ex)
   在 MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket()
   在 MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.Authenticate(Boolean reset)
   在 MySql.Data.MySqlClient.NativeDriver.Open()
   在 MySql.Data.MySqlClient.Driver.Open()
   在 MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   在 MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   在 MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   在 MySql.Data.MySqlClient.MySqlPool.GetConnection()
   在 MySql.Data.MySqlClient.MySqlConnection.Open()
   在 ZyGames.Framework.Data.MySql.MySqlDataProvider.CheckConnect()
   在 ZyGames.Framework.Data.DbConnectionProvider.Initialize()
   --- 内部异常堆栈跟踪的结尾 ---
   在 ZyGames.Framework.Data.DbConnectionProvider.Initialize()
   在 ZyGames.Framework.Game.Runtime.GameEnvironment.Start(EnvironmentSetting setting, CacheSetting cacheSetting)
   在 ZyGames.Framework.Game.Runtime.RuntimeHost.OnStart()


Time:2016/7/27 9:41:44-Trace>>CheckTableSchema UserRanking error:System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 ZyGames.Framework.Data.DbConnectionProvider.CreateDbProvider(String connectKey)
   在 ZyGames.Framework.Model.EntitySchemaSet.CheckTableSchema(SchemaTable schema)


Time:2016/7/27 9:41:44-Trace>>Not found DB connection key "" of the ChatUser entity.


Time:2016/7/27 9:41:44-Trace>>CheckTableSchema GameUser error:System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 ZyGames.Framework.Data.DbConnectionProvider.CreateDbProvider(String connectKey)
   在 ZyGames.Framework.Model.EntitySchemaSet.CheckTableSchema(SchemaTable schema)


